### DISCUSSION TITLE
how to add or subtract time in nocodb i [Jun 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2489] [Discord Threads]
> 🙋‍♂️ **sidheequesdq**

**Comment by o1lab** [comment]
> 🙋‍♂️ **sidheequesdq**  
#### anyone?  
#  
> 🙋‍♂️ **wkw**  
#### <@493161649377902613>  which db r u using  
#  
> 🙋‍♂️ **wkw**  
#### Formula: ({T2} - {T1})  
![image0](https://cdn.discordapp.com/attachments/986216701127446538/986224781433118760/unknown.png)  
#  
> 🙋‍♂️ **wkw**  
#### in PG.  
#  
> 🙋‍♂️ **sidheequesdq**  
#### Thanks.  
#  
> 🙋‍♂️ **sidheequesdq**  
#### its possible get a numeric value only. like just 15.  
#  
> 🙋‍♂️ **wkw**  
#### i think not. in fact, there should be some enhancement on it as mysql would give different format  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/986216701127446538)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
